finality-aleph/src/substrate_network.rs

```diff
@@ -69,6 +69,12 @@ impl Decode for PeerId {
     }
 }
 
+impl fmt::Display for PeerId {
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        write!(f, "{}", self.0)
```

Comment:
I was thinking maybe to print this in the form of `prefix...sufix`, where both prefix and sufix are, say `8` characters only. Because currently this is quite verbose. 

@timorl what do you think?

Reply:
Technically this would usually be enough to find the related PeerId in other logs, but I'm worried that using it for different purposes would be harder (e.g. iirc there is an RPC call to mark a node as reserved and it requires the whole PeerId). So weak (since the alternative uses are unlikely) preference for the full thing being printed, but if you think this hurts the logs too much I'm fine with printing just the version you suggested.

Reply:
I guess the first `8` characters are always the same in PeerId. So maybe we could print prefix of `4` and sufix of `8`?